Gruntfile.js

```diff
@@ -1,4 +1,11 @@
 module.exports = function(grunt) {
+  function isSauceLabsAvailableInEnvironment() {
```

Comment:
Can you move this function outside the `module.exports` function block? I have a bit of an aversion against functions defined inside functions :)